Bug: Add missing Abstract / Content warning hover description in Post/Edit.php

marcusxss opened the pull request:
When editing a post the eye icon (abstract/content warning) has a hover description when creating a post but not when editing one.
It's missing because the variable isn't set in Edit.php, unlike other places where it appears.

This tiny PR copies it over from Post.php.

annando committed:
The "summary" has now a dedicated field when creating and editing posts

nupplaphil committed:
[CI] Fix releaser

Art4 committed:
install rector as standalone dev tool

Art4 committed:
Add rector config and check command

Art4 committed:
Add refactor command, fix code style

Art4 committed:
destruct list to array

Art4 committed:
Fix code style

Art4 committed:
Rename rector config file

Art4 committed:
Add license note in rector config file

Art4 committed:
let REUSE ignore the composer files in rector install folder

marcusxss committed:
Add missing Content Warning hover description in Post/Edit.php

annando commented:
Have you checked if it works to add and remove the `abstract`? I'm not totally sure if this isn't added to a dedicated field after it had been inserted.

marcusxss commented:
This PR doesn't add the button. It's already there. It just adds the same hover description it has when creating a post.

I haven't used that functionality before, but it seems to add a small header to the post, but now that you mention it it seems like it may not be possible to add or change it after a post has been created currently...?  One can see the `abstract` BBCode and its contents when editing a post, but changing it doesn't seem to have an effect. And adding one that wasn't already there doesn't have an effect either, so yes maybe it's added to a dedicated field which currently isn't updated when editing a post.

So I guess that should be fixed? But I don't know how. I just wanted to add that missing hover text. I can create an issue for it, though?

Also it seems like the preview doesn't show the abstract at all.

annando commented:
Since it doesn't work at the moment to edit this value, I suggest to removed the button on edit.

marcusxss commented:
Closing this small UI bugfix until #14019 is resolved. Made a suggestion there for a possible solution and waiting for an answer.

marcusxss committed:
Update the text on Frio's config page

marcusxss committed:
Remove size attrs from templates, except select/option

Art4 committed:
Merge branch '2025.07-rc' into merge-2025.07-rc-into-develop

Art4 committed:
change Dockerfile to use php image

Art4 committed:
Switch to image php:8.3-apache, fix code style

annando committed:
Merge pull request #15230 from Art4/merge-2025.07-rc-into-develop

Merge 2025.07-rc into develop

Art4 committed:
Update docker-compose.yml

Art4 committed:
update devcontainer to php 8.4, update pecl packages

Art4 committed:
install pinentry-curses in devcontainer

Art4 committed:
Update .devcontainer/Dockerfile

nupplaphil committed:
Merge pull request #15231 from Art4/update-devcontainer

Update devcontainer to PHP 8.4

marcusxss committed:
UI adjustments to the admin and moderation summaries

marcusxss committed:
User menu cleanup

marcusxss committed:
Add current page highlighting to frio, fix related logic

marcusxss committed:
Fix missing link to calendar in Vier, and add calendar highlighting

marcusxss committed:
Admin logs rename settings page, minor ui adjustments

Tealk committed:
add gnome colored sheme to frio

Tealk committed:
add Licence to css file

Tealk committed:
fix code_standards_check

Tealk committed:
remove space

Tealk committed:
some light link fixes

mediaformat committed:
Add Access-Control-Allow-Origin header

Added CORS header to allow requests from any origin.

Tealk committed:
remove space

Tealk committed:
fix eventcard colore

mediaformat committed:
Merge branch 'friendica:2025.07-rc' into c2s-inbox-cors

mediaformat committed:
Include getPublicInbox in CORS header for inbox response

getPublicInbox seems to correspond to the sharedInbox endpoint, so this is worth including for c2s clients

Commit:
Add mark all as read button to circles

Commit:
fixed code standards

mediaformat committed:
Merge branch 'friendica:2025.07-rc' into c2s-inbox-cors

mediaformat committed:
Remove unnecessary line in Inbox.php

Commit:
Always show all personal notifications, remove related link to toggle

annando committed:
Check to store the user defined channel cache for all posts

annando committed:
Channel posts can now be embedded into the timeline

Commit:
Switch approach to change BaseNotifications instead

tobiasd committed:
Merge branch '2025.07-rc' into stable

tobiasd committed:
extend valitity or security.txt

tobiasd committed:
bump version 2026.01

tobiasd committed:
set release date for 2026.01

tobiasd committed:
Merge branch 'stable' into develop

tobiasd committed:
added 2026.04 to CHANGELOG

tobiasd committed:
bump version 2026.04-dev

tobiasd committed:
Fix: syntax error in SMARTY template

When viewing a contact in Friendica 2026.04-dev I encountered a syntax error in the `contact_edit` template. Adding the missing `fetchoutbox` fixes it and seems what was supposed to be there without digging deeper into the calling PHP files.

nupplaphil committed:
Merge pull request #15447 from tobiasd/20260128-bug-smartytemplateerror

Fix: syntax error in SMARTY template

nupplaphil committed:
Merge pull request #15446 from annando/adduserchannel

Check to store the user defined channel cache for all posts

annando committed:
Quote check is removed

Commit:
Make contact link to global directory pass on user language

annando committed:
Merge pull request #15440 from marcusxss/personal_notifications_always_show_all

Always show all personal notifications, remove related link to toggle

annando committed:
Merge pull request #15368 from marcusxss/ui_admin_mod_summaries

Adjustments to the admin and moderation summaries (mostly Frio)

Commit:
Adjustments to some strings around photos and events

annando committed:
Merge pull request #15428 from mediaformat/c2s-inbox-cors

C2S: Access-Control-Allow-Origin header for already authorized users

MrPetovan committed:
Merge pull request #15449 from annando/no-quote-check

Quote check is removed

tobiasd committed:
Merge pull request #15448 from annando/network-channel

Channel posts can now be embedded into the timeline

loma-one committed:
Optimization of the editor toolbar and enhancement of the preview function

This pull request implements structural improvements to the editor (compose) to enhance usability and visual clarity:

Toolbar Reorganization: Buttons have been grouped into logical Bootstrap btn-group units. Media buttons (image, attachment) are now positioned on the left (pull-left), while formatting options and special features (Emoji, CW, Code) are arranged in two distinct groups on the right (pull-right).

Enhanced Preview Logic: Added a toggle functionality to the preview button. When the preview is active, the button text changes to "Close Preview" (or local equivalent) to provide a more intuitive user experience.

annando committed:
Fixes "Argument #3 ($author_id) must be of type int, null given

annando committed:
Fixes "DOMDocument::loadHTML(): Argument #1 ($source) must not be empty"

tobiasd committed:
fix missing space in DE translation

closes #15464

tobiasd committed:
Merge pull request #15467 from annando/fix-null

Fixes "Argument #3 ($author_id) must be of type int, null given

tobiasd committed:
Merge pull request #15468 from annando/loadhtml

Fixes "DOMDocument::loadHTML(): Argument #1 ($source) must not be empty"

tobiasd committed:
update DE translation

fix missing space that @annando discovered

annando committed:
Merge pull request #15469 from tobiasd/20260204-de

fix missing space in DE translation

loma-one committed:
Remove unnecessary placeholder in compose template

Remove unnecessary placeholder in compose template

annando committed:
Fixes:  Undefined array key "filter_channels"

Espionage724 committed:
Update console.php (minor typo)

- Shortened a tiny bit too

Espionage724 committed:
Update console.php

- Even shorter (script itself is named console)

tobiasd committed:
Merge pull request #15474 from Espionage724/patch-1

Update console.php (minor typo)

tobiasd committed:
Merge pull request #15472 from annando/warning

Fixes:  Undefined array key "filter_channels"

tobiasd committed:
Merge pull request #15466 from loma-one/loma-one-patch-1

Optimization of the editor toolbar and enhancement of the preview funktion

Commit:
Remove media/media posts from the user menu to simplify it

Commit:
Photos updates

Commit:
Compose styling, larger BBCode buttons

Commit:
Photos - File browser: Larger buttons to switch file types, add text

samuel-asleep committed:
Fix event participation responses not delivered to event organizers

annando committed:
Store "View" activity as "seen" marker.

annando committed:
Ensure numeric content in numeric fields

tobiasd committed:
Merge pull request #15378 from marcusxss/frio_highlight_current_section

Add current section highlighting to Frio's navbar, fix related logic

annando committed:
Fix for empty circles

tobiasd committed:
Merge pull request #15491 from annando/circles

Fix for empty circles

tobiasd committed:
Merge pull request #15490 from annando/numeric

Ensure numeric content in numeric fields

tobiasd committed:
Merge pull request #15489 from annando/view

Store "View" activity as "seen" marker

annando committed:
Issue 14583: Avoid empty entries in list of trending links

annando committed:
Issue 15456: Fix login to Bridgy Fed

tobiasd committed:
Merge pull request #15494 from annando/issue-15456

Issue 15456: Fix login to Bridgy Fed

annando committed:
Merge pull request #15482 from marcusxss/frio_theme_text_changes

Update the text on Frio's config page

annando committed:
Issue 15478: Prevent comments when blocked or is blocked

annando committed:
Add quoted post as attachment when it couldn't be fetched

annando committed:
Two slow database queries fixed

annando committed:
Respect robots.txt when fetching server data

tobiasd committed:
Merge pull request #15499 from annando/database

Two slow database queries fixed

tobiasd committed:
Merge pull request #15493 from annando/issue-14583

Issue 14583: Avoid empty entries in list of trending links

tobiasd committed:
Merge pull request #15495 from annando/issue-15478

Issue 15478: Prevent comments when blocked or is blocked

tobiasd committed:
Merge pull request #15496 from annando/quote-fix

Add quoted post as attachment when it couldn't be fetched

tobiasd committed:
Merge pull request #15369 from marcusxss/user_menu_cleanup

Simplify the UI: User menu redundancy

Commit:
Fix CS

Commit:
Frio: Make login form background color respect dark theme

samuel-asleep committed:
Refactor event participation addressing to preserve followers notification

annando committed:
Merge pull request #15504 from marcusxss/html_remove_size_attrs

Remove size attrs from templates, except select/option

samuel-asleep committed:
Add explanatory comment for event participation addressing

annando committed:
Merge pull request #15488 from samuel-asleep/develop

Fix event participation responses not delivered to event organizers

tobiasd committed:
Merge pull request #15486 from marcusxss/photos_file_browser

Photos - File browser: Larger buttons to switch file types, add text

tobiasd committed:
Merge pull request #15500 from annando/gotosocial

Respect robots.txt when fetching server data

Commit:
Fix pagination by using array_key_first/last in Channel and Network modules

annando committed:
Merge pull request #15451 from marcusxss/link_global_directory_language

Make link from contacts to global directory pass on user language

annando committed:
Merge pull request #15507 from Raroun/fix-channel-pagination

Fix pagination by using array_key_first/last in Channel and Network module

annando committed:
Merge pull request #15402 from Tealk/feature/gnome_scheme

add gnome colored sheme to frio

nupplaphil committed:
[CI] add all-in-one package for Release

annando committed:
Don't transmit "View" activities via Diaspora

Commit:
Fix bug in #15486 - photo/file browser

nupplaphil committed:
Merge pull request #15484 from marcusxss/jot_compose_styling

Compose styling, larger BBCode buttons

nupplaphil committed:
Merge pull request #15506 from marcusxss/frio_fix_login_form_bg

Frio: Make login form background color respect dark theme

nupplaphil committed:
Merge pull request #15510 from marcusxss/fix_bug_photos

Fix bug in #15486 - photo/file browser

nupplaphil committed:
Merge pull request #15509 from annando/no-view

Don't transmit "View" activities via Diaspora

nupplaphil committed:
Merge pull request #15505 from marcusxss/terminology_photos_settings

Adjustments to some strings around photos and events

annando committed:
Don't process "View"

tobiasd committed:
Merge pull request #15511 from annando/view2

Don't process "View"

tobiasd committed:
Merge pull request #15508 from nupplaphil/ci/releaser-allinone

[CI] add all-in-one package for Release

annando committed:
Fix: Prevent not nullable fields set to null

annando committed:
Don't set "mention" on view / don't display channels on "mention" timeline

Art4 committed:
fix caching problem devcontainer

Art4 committed:
update php-cs-fixer to run it on PHP 8.4

Art4 committed:
split up composer scripts for rector:run and rectify

Art4 committed:
downgrade php-cs-fixer dependencies for PHP 7.4

nupplaphil committed:
Merge pull request #15512 from marcusxss/admin_logs_minor_adjustments

Admin logs rename settings page, minor ui adjustments

nupplaphil committed:
Merge pull request #15514 from annando/database

Fix: Prevent not nullable fields set to null

nupplaphil committed:
Merge pull request #15515 from annando/mentions

Don't set "mention" on view / don't display channels on "mention" timeline

Commit:
updated messages.po

Raroun committed:
Merge branch 'develop' into circle-mark-as-read

annando committed:
Don't copy permissions when permissions are already defined

tobiasd committed:
Merge pull request #15518 from annando/permissions

Don't copy permissions when permissions are already defined

annando committed:
Additional logging for issue 15395

Commit:
Directory: Show the number of users listed (results)

nupplaphil committed:
Merge pull request #15519 from annando/vanished-quotes

Additional logging for issue 15395

nupplaphil committed:
Merge pull request #15521 from marcusxss/photos_updates_2

Small UI/UX updates to some photos and admin pages

Commit:
Use DBA::update() instead of raw SQL for marking circle posts as read

Raroun committed:
Merge branch 'develop' into circle-mark-as-read

annando committed:
Fetch missing quoted post asynchronously

annando committed:
Merge pull request #15436 from Raroun/circle-mark-as-read

Add "Mark all as read" button to circles

nupplaphil committed:
Merge pull request #15522 from annando/quote

Fetch missing quoted post asynchronously

Commit:
Pre-fill album when uploading from album view

Commit:
Add translation strings for album upload

Commit:
fix for #15520

annando committed:
Fix some "Uncaught exception in worker method execution"

nupplaphil committed:
Merge pull request #15524 from Raroun/album-upload-fix

Photo upload to album doesn't ask for album anymore when already viewing one / Fix for Redirect to empty Page when album is deleted

nupplaphil committed:
Merge pull request #15526 from annando/uncaught

Fix some "Uncaught exception in worker method execution"

Commit:
Hide the messages sidebar when it's empty

tobiasd committed:
Merge pull request #15527 from marcusxss/messages_sidebar_hide_empty

Hide the messages sidebar when its empty (no messages)

annando committed:
Fix Database issue / log failing database queries

nupplaphil committed:
Merge pull request #15530 from annando/database

Fix Database issue / log failing database queries

Commit:
Add friendly message when registration is closed

annando committed:
Merge pull request #15532 from Raroun/register-closed-info

Enhancement: Add friendly message when registration is closed

annando committed:
Item post functionality is removed from photos

annando committed:
Move some more modules into classes

Art4 committed:
chore: set php version for rector to PHP 7.4

Art4 committed:
chore: fix code style

Art4 committed:
chore: remove @PSR1 and @PSR2 ruleset

they are part of the @PSR12 ruleset

Tealk committed:
fix colorbox theme

Art4 committed:
chore: set indent to tabs for rector

Art4 committed:
Apply suggestions from code review

Art4 committed:
chore: set rector downgrade sets to PHP 7.4

Art4 committed:
fix: downgrade str_ends_with() for PHP 7.4 support

Art4 committed:
chore: replace deprecated visibility_required rule with modifier_keywords

annando committed:
Merge pull request #15537 from Art4/fix-php-7.4-support

fix: downgrade str_ends_with() for PHP 7.4 support

annando committed:
Merge pull request #14989 from Art4/destruct-list-to-array

Destruct list() to array [Rector]

annando committed:
Merge pull request #15536 from Tealk/theme/frio/gnome/colorbox

fix colorbox theme

Art4 committed:
chore: update PSR12 to PER-CS3x0 ruleset

Art4 committed:
chore: remove php-cs-fixer rules already defined in PER-CS3.0 ruleset

Art4 committed:
chore: remove spaces_inside_parentheses rule

Art4 committed:
chore: fix code style with spaces_inside_parentheses rule

Art4 committed:
chore: fix code style

Art4 committed:
chore: add the php-cs-fixer.dist.php itself and let it fix the code style

Art4 committed:
chore: add trailing_comma_in_multiline rule with PHP 7.4 compatible rule

Art4 committed:
chore: fix code style for PHP 7.4

Art4 committed:
refactor: activate TernaryToElvisRector

Art4 committed:
refactor: run LongArrayToShortArrayRector

Art4 committed:
refactor: upgrade rector php level to 9

Art4 committed:
refactor: switch to PHP 7.4 compatible syntax without named arguments

Art4 committed:
chore: fix code style

Art4 committed:
fix: revert changes in .rector.php

randompenguin1 committed:
fixes for sortable lists

Removed mobile drag-n-drop script that did not work
Added button interface for moving list items
Fixed duplication error for keyboard control
Added animation to make it clear which way item moved

randompenguin1 committed:
Updated instruction text for sortable list

randompenguin1 committed:
Updated instruction text for sortable list

randompenguin1 committed:
removed extra single quote

Art4 committed:
chore: update rector to codebase for PHP 5.5

Art4 committed:
refactor: run ClassConstantToSelfClassRector and StringClassNameToClassConstantRector

Art4 committed:
chore: fix code style

Tealk committed:
fix disabled & readonly form-control

nupplaphil committed:
Merge pull request #15538 from Art4/review-code-style-rules

Review code style rules, change spaces_inside_parentheses rule

Art4 committed:
Merge branch 'develop' into update-codebase-to-php-5.4

nupplaphil committed:
Merge pull request #15544 from Tealk/theme/frio/gnome/form-control

fix disabled & readonly form-control

nupplaphil committed:
Merge pull request #15541 from randompenguin1/patch-1

fixes for sortable lists

Commit:
Update messages, code style

annando committed:
Update mod/photos.php

Co-authored-by: Philipp <admin+Github@philipp.info>

annando committed:
Changes after review

nupplaphil committed:
Merge pull request #15533 from annando/no-photo-post

Item post functionality is removed from photos

Art4 committed:
chore: set required php version to ^7.4 || ^8.0

Art4 committed:
chore: install php-cs-fixer in project comoser.json

Art4 committed:
chore: install rector in project comoser.json

Art4 committed:
chore: remove rector and php-cs-fixer from bin/dev folder

Art4 committed:
chore: fix composer scripts, change all php-cs-fixer and rector paths

Art4 committed:
chore: update PHPStan to 2.1.39

Art4 committed:
chore: update phpstan and php-mock with all dependencies

Art4 committed:
fix: fix phpstan errors

nupplaphil committed:
Merge pull request #15542 from randompenguin1/develop

Updated instruction text for sortable lists

Commit:
Show member since in vcard, if enabled

annando committed:
Merge pull request #15550 from marcusxss/vcard_member_since

Show "member since" in vcard, if enabled

nupplaphil committed:
Merge pull request #15534 from annando/modules

Move some more modules into classes

Art4 committed:
Merge branch 'develop' into review-composer-dependencies

Art4 committed:
chore: fix code style

nupplaphil committed:
Merge pull request #15547 from Art4/review-composer-dependencies

Move dev libraries into project composer.json

annando committed:
Additional database logging when inserting data

annando committed:
Fix code standards

nupplaphil committed:
Merge pull request #15552 from annando/db-logging

Additional database logging when inserting data

Tealk committed:
fix codeblock colore

nupplaphil committed:
Merge pull request #15554 from Tealk/theme/frio/gnome/codeblock

fix codeblock colore

annando committed:
Issue 15553: Display dates in a language depending format

annando committed:
Fix code style

annando committed:
No salmon anymore

annando committed:
API: verify_credentials can be called with any scope

annando committed:
Merge pull request #15539 from Art4/update-codebase-to-php-5.4

Update codebase to php 5.4

nupplaphil committed:
Merge pull request #15558 from annando/no-salmon

No salmon anymore

nupplaphil committed:
Merge pull request #15543 from Art4/update-codebase-to-php-5.5

Update codebase to PHP 5.5

tobiasd committed:
Merge pull request #15557 from annando/any-scope

API: verify_credentials can be called with any scope

tobiasd committed:
Merge pull request #15556 from annando/issue-15553

Issue 15553: Display dates in a language depending format

annando committed:
More localisated dates

annando committed:
Fix codestyle

tobiasd committed:
Merge pull request #15560 from annando/more-localisation

More localisated dates

annando committed:
Allow to enter birthdays on February 29th

annando committed:
Fix code standards

Commit:
Update the date format for member since/joined

nupplaphil committed:
Merge pull request #15561 from annando/leap-year

Allow entries for birthdays on 29 February

annando committed:
Merge pull request #15565 from marcusxss/format_date_joined_member_since

Update the date format for member since/joined

annando committed:
Merge pull request #15563 from marcusxss/directory_show_count_users_listed

Directory: Show the number of users listed (results)

MrPetovan committed:
Merge pull request #14019 from annando/edit-summary

The "summary" has now a dedicated field when creating and editing posts
